deployments/dockerfiles/safety/Dockerfile

```diff
@@ -1,17 +1,25 @@
 # Dockerfile used to create "husyci/safety" image
 # https://hub.docker.com/r/huskyci/safety/
 
-FROM python:3.6-alpine
+FROM python:3.6-alpine as builder
 
+# Copy JSON Values from Safety
 COPY ./script.sh /
 RUN chmod +x /script.sh
```

Comment:
@localleon thanks for your contribution! The Dockerfile works fine but you don't need those 2 lines here.

Reply:
I wasn't sure if these values are also needed in the build process. Now this is clear. I updated my PR to remove these lines